test/unit/alignment/pairwise/fixture/semi_global_affine_banded.hpp

```diff
@@ -0,0 +1,207 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+#pragma once
+
+#include <vector>
+
+#include <seqan3/alignment/configuration/align_config_aligned_ends.hpp>
+#include <seqan3/alignment/configuration/align_config_band.hpp>
+#include <seqan3/alignment/configuration/align_config_gap.hpp>
+#include <seqan3/alignment/configuration/align_config_mode.hpp>
+#include <seqan3/alignment/configuration/align_config_scoring.hpp>
+#include <seqan3/alignment/scoring/aminoacid_scoring_scheme.hpp>
+#include <seqan3/alignment/scoring/nucleotide_scoring_scheme.hpp>
+#include <seqan3/alphabet/aminoacid/aa27.hpp>
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/alphabet/nucleotide/dna5.hpp>
+
+#include "alignment_fixture.hpp"
+
+namespace seqan3::fixture::semi_global::affine::banded
```

Comment:
:open_mouth:

Reply:
it makes debugging the tests quite simple.

Reply:
Or not debugging, but you see from the gtest error output which functions failed. And there will be a lot more of tests added in the future.

Reply:
You really need a 5-fold nested namespace within the *test framework*?

Reply:
Yes.

Reply:
Can you at least put a `::test` before it to make clear that this is not part of the library?

Reply:
Well, if it makes the context more clear, why not.